.github/workflows/end-to-end-test-portals.yml

```diff
@@ -0,0 +1,86 @@
+name: End to end testing for Portals
+
+on:
+  push:
+  pull_request:
+    branches: [main]
+env:
+  PW_ALL_BLOBS_DIR: all-blob-reports
+  PORTALS_SUBDIR: apps/portals
+
+jobs:
+  e2e-tests:
+    if: ${{ github.ref_name == 'main' || github.actor == github.repository_owner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        PORTAL:
+          [
+            adknowledgeportal,
+            arkportal,
+            bsmn,
+            cancercomplexity,
+            challengeportal,
+            digitalhealth,
+            elportal,
+            nf,
+            psychencode,
+            stopadportal,
+          ]
```

Comment:
Currently runs all e2e tests against all Portals at the same time, since the jobs are relatively short (4-7 minutes / Portal). If the e2e job duration increases in the future, could set `max-parallel` to restrict the number of jobs started at a time.